integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -570,7 +568,7 @@ void testLogHomeChangeEvents() {
 
 
   /**
-   * The test modifies the includeServerOutInPodLog property and verifies the domain roll starting events are logged.
+   * The test modifies the erOutInPodLog property and verifies the domain roll starting events are logged.
```

Comment:
I'm guessing that this was an inadvertent change.

Reply:
yup, fixed.